incubator/nodejs/image/project/Dockerfile

```diff
@@ -7,11 +7,18 @@ RUN apt-get update \
  && apt-get clean \
  && echo 'Finished installing dependencies'
 
+# Copying individual files/folders as buildah 1.9.0 does not honour .dockerignore
+COPY --chown=node:node package*.json /project/
```

Comment:
These package files don't seem to be used (there is no `WORKDIR /project; RUN npm install`). The old code used to do `COPY --chown=node:node . /project`, so it would be copying `COPY package.* node_modules /project`, but that doesn't seem to be replaced. Perhaps that happens elsewhere? Sorry, I only have a general sense of how appsody uses this file.

Reply:
The intent is to copy the contents of /project and /project/user-app but not copy any `/node_modules`. 

Given /project contains files from the stack, I can achieve this by copying selective files. User's appl could be in any structure so I'm copying the entire user-app folder and then delete the user-app/node_modules folder later.

That said I agree that we are not using `/project/package.json` at all. It is only used during development and adds appmetrics-dash. So we can skip copying it here.